presto-main/src/test/java/com/facebook/presto/type/setdigest/TestMakeSetDigest.java

```diff
@@ -0,0 +1,159 @@
+        assert 2 == Integer.parseInt(assertions.getQueryRunner().execute(
+                "SELECT cardinality(make_set_digest(value)) FROM (VALUES -398741129664271, 9000000125356546) T(value)").getMaterializedRows().get(0).getField(0).toString());
+        assert "1.0".equals(assertions.getQueryRunner().execute(
+                "SELECT jaccard_index(make_set_digest(value), make_set_digest(value1)) FROM (VALUES (-398741129664271,9000000125356546),(9000000125356546,-398741129664271)) T(value,value1)").getMaterializedRows().get(0).getField(0).toString());
+        String hashCount = "{" + Murmur3Hash128.hash64(-398741129664271L) + "=1, " + Murmur3Hash128.hash64(9000000125356546L) + "=1}";
+    }
+
+    @Test
+    public void testMakeSetDigestFloat()
+    {
+        assert assertions.getQueryRunner().execute(
+                        "SELECT make_set_digest(value) " +
+                                "FROM (VALUES 1.2, 2.3) T(value)").getMaterializedRows().get(0).getField(0).toString().replaceAll("\n", "   ")
+                .equals("01 0c 00 00 00 02 0b 02   00 00 fd db 53 80 bb 20   9d 00 20 00 00 02 00 00   00 f6 96 17 64 b0 bb 20   9d 19 15 c6 ec 29 fd db   53 01 00 01 00");
+        assert 2 == Integer.parseInt(assertions.getQueryRunner().execute(
+                "SELECT cardinality(make_set_digest(value)) FROM (VALUES 1.2, 2.3) T(value)").getMaterializedRows().get(0).getField(0).toString());
+        assert "1.0".equals(assertions.getQueryRunner().execute(
+                "SELECT jaccard_index(make_set_digest(value), make_set_digest(value1)) FROM (VALUES (1.2, 2.3),(2.3, 1.2)) T(value,value1)").getMaterializedRows().get(0).getField(0).toString());
+
+        String hashCount = "{" + Murmur3Hash128.hash64(23) + "=1, " + Murmur3Hash128.hash64(12) + "=1}";
+    }
+
+    @Test
+    public void testMakeSetDigestChar()
+    {
+        assert assertions.getQueryRunner().execute(
+                        "SELECT make_set_digest(value) " +
+                                "FROM (VALUES 'c') T(value)").getMaterializedRows().get(0).getField(0).toString().replaceAll("\n", "   ")
+                .equals("01 08 00 00 00 02 0b 01   00 40 df 38 8e 00 20 00   00 01 00 00 00 d7 74 1f   4a 6c df 38 8e 01 00");
+        assert 1 == Integer.parseInt(assertions.getQueryRunner().execute(
+                "SELECT cardinality(make_set_digest(value)) FROM (VALUES 'c') T(value)").getMaterializedRows().get(0).getField(0).toString());
+        assert "0.0".equals(assertions.getQueryRunner().execute(
+                "SELECT jaccard_index(make_set_digest(value), make_set_digest(value1)) FROM (VALUES ('a', 'b'),('c', 'd')) T(value,value1)").getMaterializedRows().get(0).getField(0).toString());
+
+        String hashCount = "{" + Murmur3Hash128.hash64(Slices.utf8Slice("c")) + "=1}";
+    }
+
+    @Test
+    public void testMakeSetDigestVarchar()
+    {
+        assert assertions.getQueryRunner().execute(
```

Comment:
Please use TestNG assertions here and everywhere else in this file.  Do not use Java assert.